Lesson2/WorkNotes.py

```diff
@@ -0,0 +1,51 @@
+poem_text = """I wandered lonely as a cloud
+That floats on high o'er vales and hills,
+When all at once I saw a crowd,
+A host, of golden daffodils;
+Beside the lake, beneath the trees,
+Fluttering and dancing in the breeze. 
+
+Continuous as the stars that shine
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance."""
+
+# task 1
+"""
+Logic: 
+1) Create a list of vowels
+2) Iterate over the text and count each vowel
+3) Print the result
+"""
+vowels = ('a', 'e', 'i', 'o', 'u')
+table_style = (32, 12)
```

Comment:
Better to avoid such constructions cause not clear what we store here . For example you can use 
global_width, row_width = 32, 12   or as minimum leave comment line above.